tests/integration/Makefile

```diff
@@ -0,0 +1,38 @@
+GO_LDFLAGS := -s -w $(patsubst %,-X %, $(GO_BUILD_VARS))
+
+.PHONY: test
+test:
+	go test -v ./pkg/... -coverprofile cover.out
+
+.GOLANGCILINT_VERSION := v1.51.2
+.GOLANGCILINT_PATH := $(shell go env GOPATH)/bin/golangci-lint/$(.GOLANGCILINT_VERSION)
+
+${.GOLANGCILINT_PATH}/golangci-lint: 
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
+			| sh -s -- -b ${.GOLANGCILINT_PATH} ${.GOLANGCILINT_VERSION}
+
+.PHONY: lint
+lint: ${.GOLANGCILINT_PATH}/golangci-lint
```

Comment:
not sure I follow this is the same as other Makefiles

Reply:
There are `.golangci.yml` and `.errcheck_excludes.txt` files for each of `scheduler`, `operator`, `components/tls`, and `hodometer`.  These are configs used for linting.

I noticed because some of the integration test files haven't had `gci` run on them.

Reply:
ok will add and rerun

Reply:
https://github.com/SeldonIO/seldon-core/pull/4895/commits/2f06bdfc165c3b017dc73f3faf9bb0920ab48f25